velox/functions/prestosql/registration/DateTimeFunctionsRegistration.cpp

```diff
@@ -60,6 +63,10 @@ void registerSimpleFunctions() {
   registerFunction<YearOfWeekFunction, int64_t, Date>({"yow", "year_of_week"});
   registerFunction<YearOfWeekFunction, int64_t, TimestampWithTimezone>(
       {"yow", "year_of_week"});
+  registerFunction<WeekFunction, int64_t, Timestamp>({"week_of_year"});
+  registerFunction<WeekFunction, int64_t, Date>({"week_of_year"});
+  registerFunction<WeekFunction, int64_t, TimestampWithTimezone>(
+      {"week_of_year"});
```

Comment:
is there a short name `woy` similar to `yow` above?

Reply:
Unfortunately not, Presto documentation (https://prestodb.io/docs/current/functions/datetime.html#week) doesn't have any such function.